draft-ietf-rats-architecture.md

```diff
@@ -600,24 +609,18 @@ if the Relying Party is a constrained node.
 ~~~~
 {: #backgroundcheck title="Background-Check Model"}
 
-The background-check model is so named because of the resemblance of how employers and volunteer
-organizations perform background checks. When a prospective employee provides claims about
-education or previous experience, the employer will contact the respective institutions or
-former employers to validate the claim. Volunteer organizations often perform police background
-checks on volunteers in order to determine the volunteer's trustworthiness.
-Thus, in this analogy, a prospective volunteer is an Attester, the organization is the Relying Party,
-and a former employer or government agency that issues a report is a Verifier.
-
 ## Combinations
 
 One variation of the background-check model is where the Relying Party
-and the Verifier on the same machine, and so there is no need for a protocol between the two.
+and the Verifier are on the same machine, performing both functions together.  
+In this case, there is no need for a protocol between the two.
 
-It is also worth pointing out that the choice of model is generally up to the Relying Party,
-and the same device may need to create Evidence for different Relying Parties and different use cases
-(e.g., a network infrastructure device to gain access to the network, and then a
-server holding confidential data to get access to that data).  As such, both models may
-simultaneously be in use by the same device.
+It is also worth pointing out that the choice of model is generally up to the
+Relying Party.  The same device may need to create Evidence for different
+Relying Parties and/or different use cases.  For instance, it would provide
+Evidence to a network infrastructure device to gain access to the network, and
+to a server holding confidential data to gain access to that data. As such,
+both models may simultaneously be in use by the same device.
```

Comment:
```suggestion
It is also worth pointing out that the choice of model is generally up to the Relying Party.  
The same device may need to create Evidence for different Relying Parties and/or different use cases.  
For instance, it would provide Evidence to a network infrastructure device to gain access to the network, and
to a server holding confidential data to gain access to that data. 
As such, both models may simultaneously be in use by the same device.
```